src/profile-logic/import/chrome.js

```diff
@@ -676,6 +676,13 @@ async function processTracingEvents(
         }
       }
 
+      if (profileEvent.name === 'CpuProfile') {
+        profile.meta.profilingStartTime =
+          profileEvent.args.data.cpuProfile.startTime / 1000;
+        profile.meta.profilingEndTime =
+          profileEvent.args.data.cpuProfile.endTime / 1000;
```

Comment:
Can you please add a comment to the CpuProfileData type above, with the unit for these 2 values? They are microseconds, right?

Reply:
> Can you please add a comment to the CpuProfileData type above, with the unit for these 2 values?

Okay fdeb95268494c227edb67fbc392619723e8681ae

> They are microseconds, right?

Apparently so, based on https://stackoverflow.com/questions/53999609/understanding-javascript-cpu-profiles that links to https://chromedevtools.github.io/devtools-protocol/tot/Profiler/#type-Profile.